HW04/vitaly_kadyrko/spec/mentor_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require_relative '../lib/homework'
+require_relative '../lib/mentor'
+require_relative '../lib/student'
+require_relative '../lib/notification'
+
+RSpec.describe Mentor do
+  let(:student) { Student.new(name: 'Test', surname: 'Student') }
+  let(:mentor) { described_class.new(name: 'Test', surname: 'Mentor') }
+  let(:homework) { desribed_class.add_homework(title: 'Test title', description: 'Test description', student: student) }
+
+  desribe '#add_homework' do
+    subject { desribed_class.add_homework(title: 'Test title', description: 'Test description', student: student) }
+
+    it 'returns Homework class object' do
+      expect(subject).to be_an_instance_of(Homework)
+    end
+
+    it 'adds new_homework_message to student notifications array' do
+      expect(student.notifications[0]).to eq "New homework '#{homework.title}' was added"
+    end
+
+    it 'adds new homework to student homeworks array' do
+      expect(student.homeworks[0]).to eq homework
+    end
```

Comment:
where di you call your `subject` ? you can't just expect something, you must perform some action